src/k4a_calibration_transform_data.cpp

```
@@ -24,6 +24,22 @@ void K4ACalibrationTransformData::initialize(const k4a::device &device,
                                              const K4AROSDeviceParams params)
 {
     k4a_calibration_ = device.get_calibration(depth_mode, resolution);
+    initialize(params);
+}
+
+void K4ACalibrationTransformData::initialize(const k4a_playback_t &k4a_playback_handle,
+                                             const K4AROSDeviceParams params)
+{
+    if (k4a_playback_get_calibration(k4a_playback_handle, &k4a_calibration_) != K4A_RESULT_SUCCEEDED)
```

k4a C++ wrappers all throw exceptions on failure, which is why there is no error handling here. This should do more than just return on error: it should throw an exception to cause the node to crash, since we cannot continue without calibration data.

---

I think its fixed. I throw an std::runtime_error now.
That should resolve this right?

---

Looks good.